photo/migrations/0004_alter_contest_updated_by_alter_submission_content.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.1 on 2022-10-20 14:49
+
+from django.db import migrations, models
+import photo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("photo", "0003_comment_created_at_comment_updated_at_and_more"),
+    ]
+
+    operations = [
```

Comment:
I need to make a migrations when i'm in this branch, I think this migration file should be updated with all migrations needed at the moment.

Reply:
added a new migration file that should reflect the latest changes